admin_guide/out_of_resource_handling.adoc

```diff
@@ -0,0 +1,337 @@
+[[admin-guide-out-of-resource-handling]]
+= Out of Resource Handling
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+The node must preserve node stability when available compute resources are
+low. This is especially important when dealing with incompressible resources
+such as memory or disk. If either resource is exhausted, the node becomes
+unstable.
+
+[[out-of-resource-eviction-policy]]
+== Eviction Policy
+
+The node can proactively monitor for and prevent against total starvation of a
+compute resource. In cases where this appears to occur, the node can
+proactively fail one or more pods in order to reclaim the starved resource.
+When the node fails a pod, it terminates all containers in the pod, and the
+`*PodPhase*` is transitioned to *Failed*.
+
+Platform administrators can configure eviction settings within the
+*_node-config.yaml_* file:
+
+====
+----
```

Comment:
@derekwaynecarr can you please confirm that this syntax looks okay? Decided to note earlier on in the doc how to configure the settings.